src/ol/renderer/layerrenderer.js

```diff
@@ -191,7 +195,8 @@ ol.renderer.Layer.prototype.scheduleExpireCache =
              */
             function(tileSource, map, frameState) {
               var tileSourceKey = goog.getUid(tileSource).toString();
-              tileSource.expireCache(frameState.usedTiles[tileSourceKey]);
+              tileSource.expireCache(frameState.viewState.projection,
+                                     frameState.usedTiles[tileSourceKey]);
```

Comment:
Are you sure about this? Looking at the code (https://github.com/openlayers/ol3/blob/master/src/ol/tilecache.js#L42) I don't think `ol.TileCache#expireCache` removes tiles only if the cache is over the high water mark (no matter what is passed as `usedTiles`). Am I missing something here?